src/Makefile.am

```diff
@@ -99,7 +99,10 @@ BITCOIN_CORE_H = \
   compressor.h \
   consensus/consensus.h \
   consensus/params.h \
+  consensus/blockruleindex.h \
```

Comment:
Is this just an in-memory cache that stores nothing in disk? Then this doesn't apply.